vhdbuilder/packer/components.json

```diff
@@ -206,7 +207,9 @@
         "1.12.10-2",
         "1.12.10-3",
         "1.13.5-1",
-        "1.13.5-3"
+        "1.13.5-3",
+        "1.13.10",
```

Comment:
cilium-agent just needs 1.13.10-1, so I think we can remove 1.13.10

Reply:
addressed in the latest commit